components/esp_lvgl_port/CMakeLists.txt

```diff
@@ -1,29 +1,61 @@
-file(GLOB_RECURSE IMAGE_SOURCES images/*.c)
 
-idf_component_register(SRCS "esp_lvgl_port.c" ${IMAGE_SOURCES} INCLUDE_DIRS "include" REQUIRES "esp_lcd" PRIV_REQUIRES "esp_timer")
+#Get LVGL version
+idf_component_get_property(lvgl_ver lvgl__lvgl COMPONENT_VERSION)
```

Comment:
To avoid problems in the future: 
We should check for both `lvgl__lvgl` and `lvgl` in case someone wants to use local copy of LVGL